src/site/antora/modules/ROOT/pages/manual/configuration.adoc

```diff
@@ -14,1856 +14,1024 @@
 
-As of version 2.9, for security reasons, Log4j does not process DTD in
-XML files. If you want to split the configuration in multiple files, use
-link:#XInclude[XInclude] or link:#CompositeConfiguration[Composite
-Configuration].
+Properties::
++
+A set of reusable configuration values for property substitution.
++
+See <<property-substitution>> for details.
 
-As the previous examples have shown as well as those to follow, Log4j
-allows you to easily redefine logging behavior without needing to modify
-your application. It is possible to disable logging for certain parts of
-the application, log only when specific criteria are met such as the
-action being performed for a specific user, route output to Flume or a
-log reporting system, etc. Being able to do this requires understanding
-the syntax of the configuration files.
+Scripts::
++
+A container for JSR 223 scripts that can be used in other Log4j components.
++
+See xref:manual/scripts.adoc[Scripts configuration] for details.
 
-The configuration element in the XML file accepts several attributes:
+[id=global-configuration-attributes]
+=== Global configuration attributes
 
-[cols="1m,5a"]
+The main `Configuration` element has a set of attributes that can be used to tune the way the configuration file itself is used.
+The principal attributes are listed below.
+See xref:plugin-reference.adoc#org-apache-logging-log4j_log4j-core_org-apache-logging-log4j-core-config-Configuration[Plugin reference] for a complete list.
+
+[id=configuration-attribute-monitorInterval]
+==== [[AutomaticReconfiguration]] `monitorInterval`
+
+[cols="1h,5"]
+|===
+| Type          | `int`
+| Default value | `0`
+|===
+
+Determines the polling interval used by Log4j to check for changes to the configuration file.
+
+If a change in the configuration file is detected, Log4j automatically reconfigures the logger context.
+
+If set to `0`, polling is disabled.
+
+[CAUTION]
+====
+Unlike other logging backends Log4j Core is designed with **audit** logging in mind.
+
+During a reconfiguration process, no messages are lost.
+
+Unless the new configuration file removes an appender, the old one continues to work without interruption.
+
+Since some appender options (like the append mode used to open a file) can not be changed without closing the resource used by an appender, some attribute changes will be **ignored** during a reconfiguration.
+
+In order to modify these options during a reconfiguration you also need to change the resource used by the appender (e.g. the file name used by a file appender).
+====
+
+[id=configuration-attribute-status]
+==== `status`
+
+[cols="1h,5"]
+|===
+| Type          | link:../javadoc/log4j-api/org/apache/logging/log4j/Level.html[LEVEL]
+| Status | **DEPRECATED**
+| Default value (since 2.24.0) | xref:manual/systemproperties.adoc#log4j2.statusLoggerLevel[log4j2.statusLoggerLevel]
+| Default value (before 2.24.0) | value of `log4j2.defaultStatusLevel`
```

Comment:
As is clearly visible in this snippet, it is impossible to maintain
```
| row-1col1    | row1-col2
| row2-col...1 | row2-col...2
```
structure, hence why I advocate for
```
| row1-col1
| row1-col2

| row2-col...2
| row2-col...2
```
structure.

Reply:
The initial structure was copied from your suggestion in [#2552](https://github.com/apache/logging-log4j2/pull/2552#pullrequestreview-2032495268).

Maybe let's keep the shorter version if it fits and switch to the expanded on if it doesn't?